src/arkit.cs

```diff
@@ -509,6 +518,106 @@ interface ARSCNDebugOptions {
 		[Field ("ARSCNDebugOptionShowFeaturePoints")]
 		SCNDebugOptions ShowFeaturePoints { get; }
 	}
+
+	[iOS (11,0)]
+	[NoWatch, NoTV, NoMac]
+	[Protocol]
+	interface ARTrackable {
+		[Abstract]
+		[Export ("isTracked")]
+		bool IsTracked { get; }
+	}
+
+	[iOS (11,0)]
+	[NoWatch, NoTV, NoMac]
+	[BaseType (typeof(ARConfiguration))]
+	interface ARFaceTrackingConfiguration {}
+
+	[iOS (11,0)]
+	[NoWatch, NoTV, NoMac]
+	[BaseType (typeof(ARAnchor))]
+	interface ARFaceAnchor : ARTrackable {
+		[Export ("geometry")]
+		ARFaceGeometry Geometry { get; }
+
+		/* Will come in next PR
+		// @property (readonly, nonatomic) NSDictionary<ARBlendShapeLocation,NSNumber *> * _Nonnull blendShapes;
+		[Export ("blendShapes")]
+		NSDictionary<NSString, NSNumber> BlendShapes { get; }
+		*/
+	}
+
+	[iOS (11,0)]
+	[NoWatch, NoTV, NoMac]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface ARFaceGeometry : NSCopying {
+		/* Will come in next PR
+		[Export ("initWithBlendShapes:")]
+		IntPtr Constructor (NSDictionary<NSString, NSNumber> blendShapes);
+		*/
+
+		[Export ("vertexCount")]
+		nuint VertexCount { get; }
+
+		// @property (readonly, nonatomic) const vector_float3 * _Nonnull vertices;
```

Comment:
This comment can be removed.

Reply:
Yup missed. I'll remove in next PR.

Reply:
@rolfbjarne that's minor though, not PR blocking, I'll fix in next PR.